usb_protocol/types/descriptors/uac3.py

```diff
@@ -2,7 +2,12 @@
 # This file is part of usb-protocol.
 #
 """
-    descriptors specific to USB version 2 
+    Descriptors for USB Audio Class Devices (UAC), Release 3
+
+    [Audio30] refers to "Universal Serial Bus Device Class Definition for Audio Devices", Release 3.0, September 22, 2016
+    [Frmts30] refers to "Universal Serial Bus Device Class Definition for Audio Data Formats", Release 3.0, September 22, 2016
+    [TermT30] refers to "Universal Serial Bus Device Class Definition for Terminal Types", Release 3.0, May 31, 2006
+
     NOTE: This is not complete yet and will be extended as needed
```

Comment:
Is this comment still valid? It looks like you really fleshed the implementation out a bit. What would be missing?

Reply:
Yes, there are still a lot of descriptors missing, e.g. like the FeatureUnit.

Reply:
OK. Thanks!